packages/@react-aria/tabs/src/useTabList.ts

```diff
@@ -51,7 +52,9 @@ export function useTabList<T>(props: AriaTabListProps<T>, state: TabListState<T>
     selectionManager: manager,
     keyboardDelegate: delegate,
     selectOnFocus: keyboardActivation === 'automatic',
-    disallowEmptySelection: true
+    disallowEmptySelection: true,
+    isVirtualized,
+    scrollRef: ref
```

Comment:
Opinions on this? I added this here because end users could make their own TabList scrollable instead of following our collapse behavior